google/cloud/bigtable/table.py

```diff
@@ -615,6 +640,10 @@ def yield_rows(self, **kwargs):
         :param row_set: (Optional) The row set containing multiple row keys and
                         row_ranges.
 
```

Comment:
Do we also need to add attempt_timeout here?

Reply:
yep, done!